diff --git a/lib/components/Component/ComponentNode/ComponentWidget.js b/lib/components/Component/ComponentNode/ComponentWidget.js
index fbe4a4585944d9c8fe4128b3d48aca7916882cd6..38a330a6382b8daa3f576e91ef5502c8f3335675 100644
--- a/lib/components/Component/ComponentNode/ComponentWidget.js
+++ b/lib/components/Component/ComponentNode/ComponentWidget.js
@@ -101,7 +101,7 @@ export function ComponentWidget(props) {
     };
     return (React.createElement(ComponentNode, { previewMode: previewMode, onMouseOver: handleMouseEnter, onMouseLeave: handleMouseLeave, onDoubleClick: handleOnWidgetDoubleClick, onContextMenu: handleOnContextMenu },
         React.createElement(ComponentHeadWidget, { engine: engine, node: node, isSelected: node.getID() === selectedNodeId || node.isNodeSelected(selectedLink, node.getID()), isFocused: node.getID() === focusedNodeId || isHovered, menuItems: componentMenu, onFocusOut: handleMouseLeave }),
-        React.createElement(Tooltip, { title: displayName, placement: "bottom", enterNextDelay: 500, arrow: true }, !previewMode ? React.createElement(ComponentName, { disabled: isDisabled }, displayName) : React.createElement(React.Fragment, null)),
+        !previewMode && React.createElement(Tooltip, { title: displayName, placement: "bottom", enterNextDelay: 500, arrow: true }, React.createElement(ComponentName, { disabled: isDisabled }, displayName)),
         React.createElement(PortsContainer, null,
             React.createElement(ComponentPortWidget, { port: node.getPort(`top-${node.getID()}`), engine: engine }),
             React.createElement(ComponentPortWidget, { port: node.getPort(`bottom-${node.getID()}`), engine: engine }))));
