diff --git a/dist/index.js b/dist/index.js
index 5b01bfd3f0f3af890f412bcd18209ee747201ce0..b7ee39b2dcf16a8736237936b84589753d124110 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -4997,7 +4997,8 @@ import { styled as styled55 } from "@mui/material/styles";
 import { jsx as jsx69, jsxs as jsxs23 } from "react/jsx-runtime";
 var StyledTrigger = styled55(IconButton6, {
   name: "MuiUserMenu",
-  slot: "Trigger"
+  slot: "Trigger",
+  shouldForwardProp: (prop) => prop !== "showName"
 })(({ theme, showName }) => __spreadValues({
   padding: theme.spacing(0.5),
   gap: theme.spacing(1)
