diff --git a/dist/cjs/lucide-react.js b/dist/cjs/lucide-react.js
index 3c3034c0c4b265cace0a871eb905a880de89f1cc..c965404f504b169fa4bc2c2a12b89ea7d005e081 100644
--- a/dist/cjs/lucide-react.js
+++ b/dist/cjs/lucide-react.js
@@ -65,11 +65,12 @@ const Icon = react.forwardRef(
       ...rest
     },
     [
-      ...iconNode.map(([tag, attrs]) => react.createElement(tag, attrs)),
+      ...iconNode.map(([tag, attrs], i) => react.createElement(tag, { key: i, ...attrs })),
       ...Array.isArray(children) ? children : [children]
     ]
   )
 );
+Icon.displayName = 'LucideIcon';
 
 const createLucideIcon = (iconName, iconNode) => {
   const Component = react.forwardRef(
diff --git a/dist/esm/Icon.js b/dist/esm/Icon.js
index 665d2b3b141dcd365caa2ef40d42f1f05e58b33a..20f4a77ba2768c43de6553963d2eda11777a6483 100644
--- a/dist/esm/Icon.js
+++ b/dist/esm/Icon.js
@@ -33,11 +33,12 @@ const Icon = forwardRef(
       ...rest
     },
     [
-      ...iconNode.map(([tag, attrs]) => createElement(tag, attrs)),
+      ...iconNode.map(([tag, attrs], i) => createElement(tag, { key: i, ...attrs })),
       ...Array.isArray(children) ? children : [children]
     ]
   )
 );
+Icon.displayName = 'LucideIcon';
 
 export { Icon as default };
 //# sourceMappingURL=Icon.js.map
